### ISSUE TITLE
Get rid of default hugo logo in tab title

### ISSUE THREAD

**Issue by mpadge** [moveability/moweb#9]

**Comment by silberzwiebel** [comment]
That's a duplicate of #7, no?

**Comment by mpadge** [comment]
no, i thought #7 was about replacing the `fa_***` auto-load of faveicon stuff in the actual page(s). This one refers specifically to the pink diamond H icon that loads into the tab name.

**Comment by silberzwiebel** [comment]
actually, #7 was about the exact same thing, as this pink diamond is called [`favicon.ico`](https://github.com/moveability/moweb/blob/master/themes/hugo-agency-theme/static/favicon.ico). Not sure what you mean with the other icons that we should replace but feel free to elaborate on #7 ;)
